index.html

```diff
@@ -168,7 +168,7 @@ <h4>Professor Taki - Admin Member</h4>
           <p>Amazing plugin developer for PE and PC, teaches people worldwide how to code PocketMine plugins! Semi-retired from coding but actively participates in idea chats. "The PHP Professor Everyone Relys On : )"</p>
           <ul>
             <li><a href="https://taki21.github.io"><i class="fa fa-globe"></i></a></li>
-            <li><a href="https://twitter.com/Professor_Taki"><i class="fa fa-twitter" ></i></a></li>
+            <li><a href="https://twitter.com/Professor_Taki"><i class="fa fa-twitter"></i></a></li>
```

Comment:
Rather than changing the styling, you've changed the formatting of the file. I'm not sure what browser you're using but changing the formatting doesn't affect the position of the images on Firefox 73.0a1 (2019-12-08) (64-bit). I could be wrong though. Please provide before and after screenshots.